### ISSUE TITLE
WebVR and OSVR on Ubuntu

### ISSUE THREAD

**Issue by davidchu2000** [OSVR/OSVR-Docs#98]
I followed the steps from https://github.com/OSVR/OSVR-Docs/blob/master/Integrating-Game-Engines/WebVR/webvr.md

For Ubuntu Firefox Nightly, I ran the samples from "https://webvr.info/samples/", but the browser tab crashed for all the samples.
On the Ubuntu, OSVR can run minimal sample test program.

Does the WebVR/OSVR integration work for Ubuntu?

**Comment by ChristophHaag** [comment]
No, as far as I know it has never worked anywhere. There's been this bug report about it: https://github.com/OSVR/OSVR-Docs/issues/83

Recently a developer (kip) has said he'll be working on osvr-rendermanager support and during this will be fixing this too.
